packages/sol-rayz/src/index.ts

```diff
@@ -1,10 +1,10 @@
-export {
-  getParsedNftAccountsByOwner,
-  createConnectionConfig,
-} from "./getParsedNftAccountsByOwner";
+export { getParsedNftAccountsByOwner, createConnectionConfig } from './getParsedNftAccountsByOwner';
+
```

Comment:
This is not good. We need to do something with linter/prettier to avoid such things

Reply:
I will check maybe it is something with my editor